airbyte-webapp/src/locales/en.json

```diff
@@ -148,6 +148,10 @@
   "connectionForm.defaultSyncMode.message": "Set how Airbyte reads from the source and writes to the destination, across all streams <b>when the option is available</b>.",
   "connectionForm.bulkEdit.cancel": "Cancel",
   "connectionForm.bulkEdit.apply": "Apply",
+  "connectionForm.nonBreakingChangesPreference.label": "Non-breaking schema updates detected*",
+  "connectionForm.nonBreakingChangesPreference.message": "When Airbyte detects that the schema of the source has changed, what would you like to do?",
```

Comment:
@andyjih @edmundito I did not see any helper text written out in the Figma.  I winged it, but it's verbose.  Suggestions on trimming this down?

Reply:
Here's what I got based on how the description for the setting above is worded:

"Set how Airbyte handles syncs when it detects a non-breaking schema change in the source"